### ISSUE TITLE
fix_gamma in BatchNorm should be False?

### ISSUE THREAD

**Issue by zhuchen03** [luoyetx/WGAN#1]
Seems no point to set it to True. You can use deconvolution operators with bias instead. So I think it should be False.

**Comment by luoyetx** [comment]
bias is not necessary for deconv as bn operator is followed. set `fix_gamma` to True makes training unstable. I didn't figure out why.

**Comment by zhuchen03** [comment]
Then BN is also unnecessary in your current implementation...

Some other major differences:
1. In DCGAN, the standard deviation of the normal distribution for initialization is 0.02. Yours is 0.002.
2. In WGAN, the optimizer is RMSProp, and the critic is updated with 5 batches before updating the generator.

I am wondering why you choose those settings in the end. Have you tried the same hyperparameters as the original paper?

**Comment by luoyetx** [comment]
I have tried default parameters setting，but the result is bad，you can try it yourself. I think the loss itself from wgan is important，the training and parameters setting should be tuned using different dataset and under different situation. Current implementation may not the same as original paper and I also didn't try the dataset from the paper，but works well on my dataset.

**Comment by zhuchen03** [comment]
Thanks. Just realized that the bedroom dataset is very huge...
